docker/Dockerfile

```diff
@@ -1,37 +1,22 @@
-# Copyright 2021 DeepMind Technologies Limited
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+# TODO Need license file
 
 ARG CUDA=11.0
-FROM nvidia/cuda:${CUDA}-cudnn8-runtime-ubuntu18.04
-# FROM directive resets ARGS, so we specify again (the value is retained if
-# previously set).
-ARG CUDA
 
-# Use bash to support string substitution.
+FROM gcr.io/deeplearning-platform-release/base-cu110
+
 SHELL ["/bin/bash", "-c"]
 
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
       build-essential \
       cmake \
-      cuda-command-line-tools-${CUDA/./-} \
+      cuda-command-line-tools-11-0 \
```

Comment:
This replacement didn't work in by build env; need to circle-back

Reply:
Cause:
https://stackoverflow.com/questions/61827259/replacing-character-in-dockerfile-variable-error-missing-in-substitution

Going with solution 2.

Reply:
Also, this: https://benkyriakou.com/posts/docker-args-empty